cvm/quantization/from_tensorflow.py

```diff
@@ -231,7 +261,8 @@ def _impl(inputs, attrs, params):
 
         if pool_type == "avg":
             # TODO: this actually should be count_include_pad False
-            pool_attr['count_include_pad'] = True
+            # pool_attr['count_include_pad'] = True
+            pool_attr['count_include_pad'] = False
```

Comment:
This value actually should be false, and then why set it true before?

Reply:
When I first see here, it was set to True, so I changed this to False as implied in the comment.

Reply:
Aha, I remembered it. The AvgPooling convention from TensorFlow refers to the transformation [source code](https://github.com/apache/incubator-tvm/blob/master/python/tvm/relay/frontend/tensorflow.py#L240) from TF into TVM's relay IR. And I changed it into True directly for the purpose that our MRT only supports count_include_pad = True at the convention design. May models that go under from_tensorflow pass not accepted by out MRT after changing it to False, that's a problem, and you can check out the models containing AvgPooling operator such as resnet_v1?

And another point is that whereas the True value is set in the relay IR transformation, the default process logic in TensorFlow is indefinite to us. We'd better research the real source code logic in TF and then examine the count_include_pad's value.

The current suggestion is back to the True default value and adds more comments in the source such as TODO.